ecologits/model_repository.py

```diff
@@ -37,10 +37,16 @@ def __init__(self, models: list[Model]) -> None:
 
     def find_model(self, provider: str, model_name: str) -> Optional[Model]:
         for model in self.__models:
-            if model.provider == provider and model_name == model.name:
+            if model.provider == provider and model.name in model_name:
```

Comment:
@cvarrei This should be reverted no?

Reply:
In fact, it helps to use the huggingface or mistral model cases with LiteLLM : 

- For Mistral, for example, we have to call the model with its name and the prefix "mistral/": ‘mistral/mistral-small'' (thus, **model.name == ‘mistral-small’ and model_name == ‘mistral/mistral-small’**)
So, _"model_name in model.name"_ will not work (‘mistral/mistral-small’ in ‘mistral-small’); however, with the slight modification I've made, it gives _‘mistral-small’ in ‘mistral/mistral-small’_. So it can work when model_name is ‘mistral/mistral-small’ or ‘mistral_small’. 
(ref: https://litellm.vercel.app/docs/providers/mistral)

- For Huggingface, it works the same way: if I want to call ‘WizardLM/WizardCoder-Python-34B-V1.0’ with LiteLLM, I have to use ‘huggingface/WizardLM/WizardCoder-Python-34B-V1.0’.

Reply:
Thank you for the modifications! 

It was a pleasure to contribute to this project! It's really important to have metrics that open our eyes to the ecological reality of using LLMs

Reply:
Smart move! Could you just maybe explain this in one line of comment next to line 40? Otherwise good to merge for me

Reply:
I've just added the comment